src/app/components/tree-view/index.html

```diff
@@ -0,0 +1,209 @@
+    </p>
+    
+    <sky-demo-page-module-info
+      skyuxModule="SkyAngularTreeModule"
+      packageName="@skyux/angular-tree-component"
+      githubRepo="skyux-angular-tree-component"
+    >
+    </sky-demo-page-module-info>
+
+    <sky-demo-page-properties
+      sectionHeading="Angular tree view components"
+    >
+      <p>
+        To apply the SKY UX themes and additional functionality to the <sky-code>angular-tree-component</sky-code>, you must include the following components.
+      </p>
+
+      <sky-demo-page-property
+        propertyName="sky-angular-tree-wrapper"
+        isOptional="false"
+      >
+        Wraps the <sky-code>tree-root</sky-code> component. For more information on <sky-code>tree-root</sky-code>, refer to <a href="https://angular2-tree.readme.io/docs">the angular-tree-component documentation.</a>
+      </sky-demo-page-property>
+      <sky-demo-page-property
+        propertyName="sky-angular-tree-node"
+        isOptional="false"
+      >
+        Replaces the default tree node template with a SKY UX node. This component must be wrapped inside a <sky-code>ng-template</sky-code> tag with the template reference variable <sky-code>#treeNodeFullTemplate</sky-code>. For more information on templating the tree nodes, refer to <a href="https://angular2-tree.readme.io/docs">the angular-tree-component documentation.</a>
+      </sky-demo-page-property>
+    </sky-demo-page-properties>
+
+    <sky-demo-page-properties
+      sectionHeading="Angular tree wrapper component properties"
+    >
+      <sky-demo-page-property
+        propertyName="readOnly"
+        isOptional="true"
+        defaultValue="false"
+      >
+        Renders the tree in read only mode, which disables any "selected" or "active" states on the nodes. This property accepts <sky-code>boolean</sky-code> values.
+      </sky-demo-page-property>
+      <sky-demo-page-property
+        propertyName="selectLeafNodesOnly"
+        isOptional="true"
+        defaultValue="false"
+      >
+        If the tree has checkboxes enabled, this property prevents parent nodes from being selected. To enable checkboxes, refer to <a href="https://angular2-tree.readme.io/docs">the angular-tree-component documentation</a>. This property accepts <sky-code>boolean</sky-code> values.
+      </sky-demo-page-property>
+      <sky-demo-page-property
+        propertyName="selectSingle"
+        isOptional="true"
+        defaultValue="false"
+      >
+        If the tree has checkboxes enabled, this property only allows one node to be selected at a time. To enable checkboxes, refer to <a href="https://angular2-tree.readme.io/docs">the angular-tree-component documentation</a>. This property accepts <sky-code>boolean</sky-code> values.
```

Comment:
Indicates whether users can select one node at a time when checkboxes are enabled. 

(Same for this one, John!)

Reply:
I reworked this in my edits at https://github.com/blackbaud/skyux2-docs/pull/710/.